tasklist/client/src/common/processes/FormModal/index.test.tsx

```diff
@@ -294,8 +297,11 @@ describe('<FormModal />', () => {
 
     expect(screen.queryByTestId('form-skeleton')).not.toBeInTheDocument();
 
-    await user.type(
+    await user.click(
```

Comment:
This happens because there's an issue with `form-js` debounced validation.
This is not necessary on a real browser